modules/formulize/admin/formindex.php

```diff
@@ -916,6 +916,17 @@ function patch31() {
   INDEX i_sid (`sid`)
 ) ENGINE=MyISAM;";
                 }
+

```

Comment:
This block executes to upgrade from a much older version of Formulize. If you move this to around line 284, then it will run.